docs/modules/gear/shops.mdx

```diff
@@ -50,7 +50,7 @@ Items can be purchased with multiple currencies using the `<payment>` tag.
 |---|---|---|---|
 | `currency` | The currency needed to purchase. | [Material Name](/docs/reference/items/inventory#material-matchers) |
 | `price` | The amount of a currency needed to purchase. | <span className="badge badge--primary">Number</span> |
-| `kit` | The kit to give to players purchasing the item. | [Kit](/docs/modules/gear/kits) |
+| `action` | The action to execute after the player purchases an item. | [Action](/docs/modules/mechanics/actions-triggers) |
```

Comment:
should note that it has to be a player-scoped action, maybe also mention that kits are a type of player-scoped action